### ISSUE TITLE
[addon-info] fails with flow union types

### ISSUE THREAD

**Issue by GuyLivni** [storybooks/storybook#2897]
### Issue details

Expected storybook to load correctly with addon info, instead it crushes with the following error:
```
Cannot read property 'length' of undefined
TypeError: Cannot read property 'length' of undefined
    at OneOfType (http://localhost:9009/static/preview.bundle.js:71199:31)
```

### Steps to reproduce

1. Start storybook with a story that uses info addon.
2. Make sure that you have flow-bin in project, and at least one of the variables typed as a union.

The error seems to come from addon-info/dist/components/types/OneOfType.js
It fails on:
https://github.com/storybooks/storybook/blob/master/addons/info/src/components/types/OneOfType.js#L7-L10

If I change `propType.value.length` to `propType.elements.length` and `propType.value.map` to `propType.elements.map` everything works as expected.

### Please specify which version of Storybook and optionally any affected addons that you're running

- storybook/react 3.3.11
- storybook/addon-info 3.3.11
- flow-bin 0.64.0

### Affected platforms 
The brake happened when I updated addon-info from 3.2.19 --> 3.3.11

### Screenshots / Screencast / Code Snippets (Optional)

```js
type Props = { value?: string | number  }
```
![screen shot 2018-02-01 at 12 18 22 pm](https://user-images.githubusercontent.com/6113581/35673427-3a978b44-074a-11e8-927a-52055161fca2.png)

**Comment by Hypnosphi** [comment]
Probably related: https://github.com/storybooks/storybook/issues/2603

You're welcome to open a PR fixing that

**Comment by GuyLivni** [comment]
Sure, thanks

**Comment by Hypnosphi** [comment]
Released as [v3.3.13](https://github.com/storybooks/storybook/releases/tag/v3.3.13)
